### ISSUE TITLE
Zoom Broken

### ISSUE THREAD

**Issue by nebrelbug** [pvorb/npm-stat.com#82]
https://npm-stat.com/charts.html?package=squirrelly&from=2017-01-01&to=2019-11-03

Downloads per day doesn't allow zoom

**Comment by pvorb** [comment]
It does for me. What browser and OS are you using?

**Comment by nebrelbug** [comment]
@pvorb Linux and Chrome. Before I was using Windows + Chrome and had the same issue

**Comment by nebrelbug** [comment]
It shows the highlight when I click and drag, but when I close click it just resets the drag start position

**Comment by nebrelbug** [comment]
@pvorb tested on Firefox as well. Note that the problem is only with the "Downloads per Day" chart, all the other charts work fine

**Comment by pvorb** [comment]
Yeah, I could also reproduce it. But I have no idea what feature highcharts relies on, that no longer works in browsers. I'll probably have to update highcharts.

**Comment by nebrelbug** [comment]
Hi @pvorb! Some additional info - the error says 
```
Uncaught TypeError: Cannot read property 'r' of undefined
    at Fa.setState (highcharts-3.0.10.min.js:293)
    at highcharts-3.0.10.min.js:148
    at Array.forEach (<anonymous>)
    at each (highcharts-3.0.10.min.js:22)
    at Q.Tooltip.refresh (highcharts-3.0.10.min.js:148)
    at Q.Pointer.runPointActions (highcharts-3.0.10.min.js:153)
    at Q.Pointer.onContainerMouseMove (highcharts-3.0.10.min.js:160)
    at HTMLDivElement.b.onmousemove (highcharts-3.0.10.min.js:162)
setState @ highcharts-3.0.10.min.js:293
(anonymous) @ highcharts-3.0.10.min.js:148
each @ highcharts-3.0.10.min.js:22
refresh @ highcharts-3.0.10.min.js:148
runPointActions @ highcharts-3.0.10.min.js:153
onContainerMouseMove @ highcharts-3.0.10.min.js:160
b.onmousemove @ highcharts-3.0.10.min.js:162
```

**Comment by vatson** [comment]
Vote for too. I came across the same behavior that @nebrelbug described.

My system specs:
OS: macOS 10.13.6
Tested browsers: Chrome 80.0.3987.149, FireFox 74, Safari 13.1 (13609.1.20.111.8) 

<img width="1037" alt="Screen Shot 2020-04-08 at 10 50 58 PM" src="https://user-images.githubusercontent.com/365720/78827256-7971d480-79eb-11ea-8a8c-029402768252.png">

**Comment by vatson** [comment]
UPD: @pvorb I found a dependency with a choice of date range over 33 months. If the user selects more than this range then the zoom stops working
